ui/admin/tests/acceptance/onboarding/success-test.js

```diff
@@ -32,18 +34,24 @@ module('Acceptance | onboarding | success', function (hooks) {
   test('check the controller url is copyable', async function (assert) {
     const origin = window.location.origin;
     await visit(urls.success);
-    assert.dom('.hds-copy-snippet__text').isVisible();
+    assert.dom(selectors.COPY_BTN_TEXT).isVisible();
     assert.strictEqual(
-      find('.hds-copy-snippet__text').textContent.trim(),
+      find(selectors.COPY_BTN_TEXT).textContent.trim(),
       origin,
     );
```

Comment:
This is an older way we would assert specific text was correct. Today, we use `assert.dom().hasText(text)`. So this would look like `assert.dom(selectors.COPY_BTN_TEXT).hasText(origin);`.